camera_to_hls/lib/membrane_demo_camera_to_hls/pipeline.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Membrane.Demo.CameraToHls.Pipeline do
+  use Membrane.Pipeline
+
+  @impl true
+  def handle_init(_) do
+    children = [
+      # Captures video from the camera (raw video, depending on camera/os)
+      source: Membrane.CameraCapture,
+      # Converts pixel format to I420 (this is still a raw video, but different format)
+      converter: %Membrane.FFmpeg.SWScale.PixelFormatConverter{format: :I420},
+      # Takes raw I420 frames and converts to H264
+      # baseline profile tells encoder not to generate
+      # B-frames because browsers can't handle them
```

Comment:
First problem: grammatical correctness. Second issue is that these sentences are not precise enough.
```suggestion
      # Takes raw video in I420 pixel format and encodes it into H264.
      # Baseline profile tells encoder not to generate
      # B-frames because browsers can't render them properly
```